### DISCUSSION TITLE
vue2.6.11中使用出错

### DISCUSSION THREAD

**Discussion by unicycle-master** [Tencent/wujie#158] [Q&A]
"wujie-vue2": "^1.0.0-rc.12"
使用：
import WujieVue from "wujie-vue2";
Vue.use(WujieVue)

报错：

 No overload matches this call.
  Overload 1 of 2, '(plugin: PluginObject<unknown> | PluginFunction<unknown>, options?: unknown): VueConstructor<Vue>', gave the following error.
    Argument of type '{ bus: EventBus; setupApp: (options: cacheOptions) => void; preloadApp: (preOptions: preOptions) => void; destroyApp: (id: string) => void; install: (Vue: Vue) => void; }' is not assignable to parameter of type 'PluginObject<unknown> | PluginFunction<unknown>'.
      Type '{ bus: EventBus; setupApp: (options: cacheOptions) => void; preloadApp: (preOptions: preOptions) => void; destroyApp: (id: string) => void; install: (Vue: Vue) => void; }' is not assignable to type 'PluginObject<unknown>'.
        Types of property 'install' are incompatible.
          Type '(Vue: Vue) => void' is not assignable to type 'PluginFunction<unknown>'.
            Types of parameters 'Vue' and 'Vue' are incompatible.
              Type 'VueConstructor<Vue>' is missing the following properties from type 'Vue': $el, $options, $parent, $root, and 37 more.
  Overload 2 of 2, '(plugin: PluginObject<any> | PluginFunction<any>, ...options: any[]): VueConstructor<Vue>', gave the following error.
    Argument of type '{ bus: EventBus; setupApp: (options: cacheOptions) => void; preloadApp: (preOptions: preOptions) => void; destroyApp: (id: string) => void; install: (Vue: Vue) => void; }' is not assignable to parameter of type 'PluginObject<any> | PluginFunction<any>'.
      Type '{ bus: EventBus; setupApp: (options: cacheOptions) => void; preloadApp: (preOptions: preOptions) => void; destroyApp: (id: string) => void; install: (Vue: Vue) => void; }' is not assignable to type 'PluginObject<any>'.
        Types of property 'install' are incompatible.
          Type '(Vue: Vue) => void' is not assignable to type 'PluginFunction<any>'.
            Types of parameters 'Vue' and 'Vue' are incompatible.
              Type 'VueConstructor<Vue>' is not assignable to type 'Vue'.

**Comment by unicycle-master** [comment]
"wujie-vue2": "^1.0.0-rc.16"    一样

**Comment by yiludege** [comment]
感觉是 typescript 出错了，可以给一个最小复现的 demo 吗？

**Reply by unicycle-master** [reply]
是typescript 的错，解决了

**Reply by harlan1012** [reply]
哪里的错？怎么解决的呢？
